### PR TITLE
Handle Profile User

### PR DESCRIPTION
**Pull request by tienquanptit**

### CODE REVIEW

File: app/Http/Controllers/User/ProfileController.php [hunk 1]
```diff
@@ -52,34 +64,80 @@ public function show($id)
...
+            'nick_name' => $request->nick_name,
+            'phone' => $request->phone,
+            'address' => $request->address,
+            'gender' => $request->gender,
+        ]);
+
+        DB::commit();
+
+        return response()->json([
+            'error'     =>  false,
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/ProfileController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ProfileController.php</br>**Line**: 104</br>Expected 1 space between "'error'" and double arrow; 5 found

Comment (framgiaciv3) [inline app/Http/Controllers/User/ProfileController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ProfileController.php</br>**Line**: 104</br>Expected 1 space between double arrow and "false"; 2 found

File: app/Http/Controllers/User/ProfileController.php [hunk 2]
```diff
@@ -52,34 +64,80 @@ public function show($id)
...
+            'message'   =>  trans('message.success'),
```
Comment (framgiaciv3) [inline app/Http/Controllers/User/ProfileController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterComma</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ProfileController.php</br>**Line**: 105</br>Expected 1 space between "'message'" and double arrow; 3 found

Comment (framgiaciv3) [inline app/Http/Controllers/User/ProfileController.php]: **Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Controllers/User/ProfileController.php</br>**Line**: 105</br>Expected 1 space between double arrow and "trans"; 2 found

File: app/Http/Controllers/User/ProfileController.php [hunk 3]
```diff
@@ -14,7 +18,15 @@ class ProfileController extends Controller
...
     {
-        return view('user.profile.index');
+        $user = Auth::id();
+        $posts = Post::with('topic')->where([
+            ['user_id', '=', $user],
+            ['type', '=', 1],
+        ])->get();
+
+
+
```
Comment (BunnyPi04) [inline app/Http/Controllers/User/ProfileController.php]: xóa dòng trắng dư

File: resources/js/user.js [hunk 4]
```diff
@@ -253,3 +253,18 @@ $(document).on('click', '.reported', function () {
...
+        var reader = new FileReader();
+        reader.onload = function(e) {
+            $('#imagePreview').css('background-image', 'url('+e.target.result +')');
+            $('#imagePreview').hide();
+            $('#imagePreview').fadeIn(650);
+        }
+        reader.readAsDataURL(input.files[0]);
+    }
+}
+$("#imageUpload").change(function() {
```
Comment (BunnyPi04) [inline resources/js/user.js]: sử dụng nháy đơn

File: resources/js/user.js [hunk 5]
```diff
@@ -253,3 +253,18 @@ $(document).on('click', '.reported', function () {
         }
     });
 });
+
+function readURL(input) {
+    if (input.files && input.files[0]) {
+        var reader = new FileReader();
+        reader.onload = function(e) {
+            $('#imagePreview').css('background-image', 'url('+e.target.result +')');
```
Comment (BunnyPi04) [inline resources/js/user.js]: convention

**Pull request merged**